nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py

```diff
@@ -20,8 +20,27 @@
     ')', '[', ']', '{', '}',
 )
 
+GRAPHEME_CHARACTER_SETS = {
+    "en-us": (
+        'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J',
+        'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T',
+        'U', 'V', 'W', 'X', 'Y', 'Z', 'À', 'É'
+    ),
+    "es": (
```

Comment:
nit: since we are expanding languages and dialects, shall we start now using standardized locale code (ISO 639-3 for the language name plus ISO 3166 for the country code)? for example, `es-ES`, `de-DE`, `en-US`?

Reply:
Using ISO does make sense. I chose these language tags because they are what eSpeak recognizes. But it makes more sense to use the official language tags, and manually map them to the eSpeak ones as needed.

Reply:
@rlangman, what's the motivation behind adding a few accented chars, like `À` into en-US set?

Reply:
In English there are a lot of words adopted from other languages with accented characters. I got this list from parsing the ipa_cmudict on main. However looking at it closer, there is only a single accented word (DÉJÀ, as in "déjà vu") and all other words which should be accented (eg. naïve, café) had their accents stripped before we included them in our dictionary. Most likely the CMU Arpabet dictionary did not have them.

For IPA I don't think there is any reason to remove these characters from English since we definitely need to support them for other languages. But supporting them requires us to add such words to our English dictionary (ideally we would include both the accented and non-accented versions).

In the meantime, I should probably either remove these 2 characters or include other accented characters too.

Happy to here anyone else's thoughts on the matter.

Reply:
For now I decided to remove those 2 accented chars and leave the option of coming up with a more complete list of chars/words for a different PR.